includes/class-sensei.php

```diff
@@ -1844,6 +1844,10 @@ public function add_editor_caps() {
 
 		if ( ! is_null( $role ) ) {
 			$role->add_cap( 'manage_sensei_grades' );
+
+			$role->add_cap( 'manage_lesson_categories' );
```

Comment:
Admin and editor weren't receiving these capabilities previously. I think we didn't notice it before because we weren't explicitly using it.